internal/mtls/ca.go

```diff
@@ -37,6 +37,7 @@ type CAProvider interface {
 	GetCACertificate() (*CertificateGroup, error)
 	CreateRegistrationCertificate(name string) (map[string][]byte, error)
 	SignCSR(CSRPem string, commonName string, expiration time.Time) ([]byte, error)
+	GetServercertificate(dnsNames []string, localhostEnabled bool) (*CertificateGroup, error)
```

Comment:
`GetServiceCertificate`

Reply:
@jakub-dzon  did you mean `GetServerCertificate` ? or to change the Server to Service?

Reply:
Of course, `GetServerCertificate`. :facepalm: